index.js

```diff
@@ -26,22 +40,31 @@ function attacher(destination, options) {
   return destination ? bridge(destination, settings) : mutate(settings)
 }
 
-// Bridge-mode.
-// Runs the destination with the new mdast tree.
+/**
+ * Bridge-mode.
+ * Runs the destination with the new mdast tree.
+ * @param {FrozenProcessor} destination
+ * @param {Settings} options
```

Comment:
I believe that options are optional here :thinking: 
```suggestion
 * @param {Settings} [options]
```